src/app/ReadClient.h

```diff
@@ -48,12 +48,12 @@
 #include <messaging/Flags.h>
 #include <protocols/Protocols.h>
 #include <system/SystemPacketBuffer.h>
-
 namespace chip {
 namespace app {
 
 class InteractionModelEngine;
 
+#if CHIP_CONFIG_ENABLE_READ_CLIENT
```

Comment:
Why is this not above outside the "namespace chip"?

Reply:
Moved outside.